client/my-sites/site-settings/delete-site-options/index.jsx

```diff
@@ -11,68 +11,62 @@ import CompactCard from 'components/card/compact';
 import DeleteSiteWarningDialog from 'my-sites/site-settings/delete-site-warning-dialog';
 import config from 'config';
 import { tracks } from 'lib/analytics';
+import { localize } from 'i18n-calypso';
+import SectionHeader from 'components/section-header';
 
 const trackDeleteSiteOption = ( option ) => {
 	tracks.recordEvent( 'calypso_settings_delete_site_options', {
 		option: option
 	} );
 };
 
-module.exports = React.createClass( {
-	displayName: 'DeleteSite',
-
-	propTypes: {
-		sitePurchases: React.PropTypes.array.isRequired,
-		hasLoadedSitePurchasesFromServer: React.PropTypes.bool.isRequired,
-		site: React.PropTypes.object.isRequired
-	},
+class SiteTools extends Component {
+	static propTypes = {
+		sitePurchases: PropTypes.array.isRequired,
+		hasLoadedSitePurchasesFromServer: PropTypes.bool.isRequired,
+		site: PropTypes.object.isRequired
+	}
 
-	getInitialState() {
-		return {
-			showDialog: false,
-			showStartOverDialog: false
-		};
-	},
+	state = {
+		showDialog: false,
+		showStartOverDialog: false,
+	}
 
 	render() {
+		const { translate } = this.props;
+
 		const selectedSite = this.props.site;
 		const changeAddressLink = `/domains/manage/${ selectedSite.slug }`;
 		const themeSetupLink = `/settings/theme-setup/${ selectedSite.slug }`;
 		const startOverLink = `/settings/start-over/${ selectedSite.slug }`;
 		const deleteSiteLink = `/settings/delete-site/${ selectedSite.slug }`;
-		let changeAddressLinkText = this.translate( 'Register a new domain or change your site\'s address.' );
-		const themeSetupLinkText = this.translate( 'Make your site look like your theme\'s demo.' );
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Automatically make your site look like your theme\'s demo.' );
 		const strings = {
-			changeSiteAddress: this.translate( 'Change Site Address' ),
-			themeSetup: this.translate( 'Theme Setup' ),
-			startOver: this.translate( 'Start Over' ),
-			deleteSite: this.translate( 'Delete Site' )
+			changeSiteAddress: translate( 'Change your site address' ),
+			themeSetup: translate( 'Theme setup' ),
+			startOver: translate( 'Delete your content' ),
+			deleteSite: translate( 'Delete your site permanently' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **80** times:
`translate( 'Delete My Site Permanently' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).